src/js/drop-down.js

```diff
@@ -14,26 +14,33 @@ class DropDown {
 		if (!this.nav) { return; }
 
 		this.burger = this.headerEl.querySelector('.o-header-services__hamburger-icon');
-		this.burger.addEventListener('click', this.toggleDropdown.bind(this));
+		this.burger.addEventListener('click', this);
+		window.addEventListener('resize', this);
+		window.addEventListener('keydown', this);
 
-		window.addEventListener('resize', oUtils.debounce(this.render.bind(this), 100));
-		window.addEventListener('keydown', (e) => {
+		this.render();
+	}
+
+	handleEvent(e) {
+		if (e.type === 'resize') {
+			oUtils.debounce(this.render(), 100);
```

Comment:
This will execute `this.render` on every resize event. What we need to do is create a debounced version of `this.render` and call that on every resize event instead. `oUtils.debounce` will take a function and a debounce time and return a debounced version of the function which is called once per debounce time. `oUtils.debounce` will call the function with an incorrect context (`this`), we can fix that by either using an arrow function or by binding the correct context to the function.